constants/roles.js

```diff
@@ -0,0 +1,6 @@
+const ROLES = {
+  SUPERUSER: 'superUser',
```

Comment:
I think the role key for SUPERUSER is super_user
Please confirm @swarajpure

Reply:
In DB, it is stored as `super_user` and `app_owner`. 

But when adding the authorization to any route using `authorizeUser`, the roles that should be passed in `superUser`.

Please check the implementation in `authorization.js`